packages/common/__test__/common-basic.test.ts

```diff
@@ -32,6 +32,8 @@ const sampleData = {
   },
   enum: 'type1',
   smallint: 12,
+  createdAt: Date.now(),
```

Comment:
This field should be auto generated rather than entered from client side, as mentioned

Reply:
Instead of this, we could check the validation object for a subset of values, then verify that the returned values of createdAt and updatedAt are valid datetimes